client/rest-high-level/src/main/java/org/elasticsearch/client/Request.java

```diff
@@ -373,12 +377,12 @@ static Request search(SearchRequest searchRequest) throws IOException {
 
     static Request searchScroll(SearchScrollRequest searchScrollRequest) throws IOException {
         HttpEntity entity = createEntity(searchScrollRequest, REQUEST_BODY_CONTENT_TYPE);
-        return new Request("GET", "/_search/scroll", Collections.emptyMap(), entity);
+        return new Request("GET", "/_search/scroll", emptyMap(), entity);
```

Comment:
I can remove this, if needed.

Reply:
yes please, I commented above about it, before seeing your question ;)